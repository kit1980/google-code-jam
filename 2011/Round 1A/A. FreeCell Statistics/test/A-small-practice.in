100
2 87 64
8 76 42
10 12 30
2 94 79
6 4 45
5 40 16
10 91 68
10 70 10
4 8 63
10 15 99
10 17 0
10 76 79
3 100 100
3 11 47
7 48 78
1 82 100
2 25 21
10 5 20
9 9 74
8 10 30
2 9 42
5 70 29
10 10 64
4 76 54
10 58 37
1 50 2
4 9 66
5 39 26
5 84 99
3 0 100
9 68 41
7 52 83
4 40 71
1 7 94
1 83 7
10 30 93
5 10 27
5 25 99
5 25 17
4 46 60
2 50 63
9 53 44
5 47 39
10 46 90
2 64 25
10 78 30
2 25 65
5 60 22
3 53 36
6 10 69
3 72 43
4 16 11
4 71 56
8 56 6
1 77 20
5 64 23
10 26 65
7 74 77
2 50 89
2 50 6
1 50 6
3 80 52
8 20 49
4 44 48
10 15 64
2 13 88
9 6 91
10 39 47
7 51 82
10 59 67
9 67 70
4 80 96
1 50 41
10 29 44
8 70 9
1 46 90
5 81 62
1 80 0
10 56 45
6 23 94
7 91 79
5 75 15
5 5 38
7 100 0
5 30 78
3 50 100
7 60 43
1 81 76
10 85 19
3 77 51
6 55 18
4 22 48
6 8 54
8 23 10
2 75 26
7 56 7
5 24 26
2 50 0
10 82 72
7 0 0
