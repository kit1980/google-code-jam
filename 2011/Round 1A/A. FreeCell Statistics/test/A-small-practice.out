Case #1: Broken
Case #2: Broken
Case #3: Broken
Case #4: Broken
Case #5: Broken
Case #6: Possible
Case #7: Broken
Case #8: Possible
Case #9: Broken
Case #10: Broken
Case #11: Broken
Case #12: Broken
Case #13: Possible
Case #14: Broken
Case #15: Broken
Case #16: Broken
Case #17: Broken
Case #18: Broken
Case #19: Broken
Case #20: Broken
Case #21: Broken
Case #22: Broken
Case #23: Possible
Case #24: Broken
Case #25: Broken
Case #26: Broken
Case #27: Broken
Case #28: Broken
Case #29: Broken
Case #30: Broken
Case #31: Broken
Case #32: Broken
Case #33: Broken
Case #34: Broken
Case #35: Broken
Case #36: Possible
Case #37: Broken
Case #38: Possible
Case #39: Possible
Case #40: Broken
Case #41: Possible
Case #42: Broken
Case #43: Broken
Case #44: Broken
Case #45: Broken
Case #46: Broken
Case #47: Broken
Case #48: Possible
Case #49: Broken
Case #50: Broken
Case #51: Broken
Case #52: Broken
Case #53: Broken
Case #54: Broken
Case #55: Broken
Case #56: Broken
Case #57: Broken
Case #58: Broken
Case #59: Possible
Case #60: Possible
Case #61: Broken
Case #62: Broken
Case #63: Possible
Case #64: Broken
Case #65: Broken
Case #66: Broken
Case #67: Broken
Case #68: Broken
Case #69: Broken
Case #70: Broken
Case #71: Broken
Case #72: Broken
Case #73: Broken
Case #74: Broken
Case #75: Broken
Case #76: Broken
Case #77: Broken
Case #78: Broken
Case #79: Broken
Case #80: Broken
Case #81: Broken
Case #82: Possible
Case #83: Broken
Case #84: Broken
Case #85: Broken
Case #86: Broken
Case #87: Possible
Case #88: Broken
Case #89: Broken
Case #90: Broken
Case #91: Broken
Case #92: Broken
Case #93: Broken
Case #94: Broken
Case #95: Broken
Case #96: Broken
Case #97: Broken
Case #98: Broken
Case #99: Broken
Case #100: Possible
